device/marvell/armhf-marvell_et6448m_52x-r0/et6448m/profile.ini

```diff
@@ -0,0 +1 @@
+switchMacAddress=00:50:43:ee:ee:ee
```

Comment:
does it mean all switches will have the same mac?

Reply:
No, but Hardware doesn't have board eeprom, so its been updated in config file. Need to be updated per board.

Reply:
I do not quite understand the logic here, the mac address should be embedded in your board, we should not put a file in the image because then every image will be different for each board. but, I will let you address this problem later.